controllers/device_list.js

```diff
@@ -2388,6 +2388,24 @@ deviceListController.getDeviceReg = function(req, res) {
   });
 };
 
+const deviceTransaction = function(
+  transactionDB, device, audit,
+  field, auditField, value, auditBool = false,
+) {
+  if (auditBool) {
+    audit[auditField] = {
+      old: device[field] === 1,
+      new: value === 1,
+    };
+  } else {
+    audit[auditField] = {
+      old: device[field],
+      new: value,
+    };
+  }
```

Comment:
`auditBool` não ficou legal não, algo como `castAuditToBool` ficaria melhor:
```suggestion
  audit[auditField] = {
    old: castAuditToBool ? device[field] === 1 : device[field],
    new: castAuditToBool ? value ==== 1 : value,
  };
```
Aproveitando, o cast pra `bool` é sempre feito comparando com 1? Pode ficar meio pegadinha, talvez se a gente usar a `utilHandlers.convertToBoolean` no lugar fique melhor?